cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go

```diff
@@ -19,7 +19,7 @@ package v1alpha1
 import "k8s.io/kubernetes/pkg/api/unversioned"
 
 type MasterConfiguration struct {
-	unversioned.TypeMeta
+	unversioned.TypeMeta `json:",inline"`
```

Comment:
No reason. Just seems redundant. This is fine though since other api groups are doing this.